DataTests/BookmarkTests.swift

```diff
@@ -9,8 +9,10 @@ import CoreData
 class BookmarkTests: CoreDataTestCase {
     let fetchRequest = NSFetchRequest<Bookmark>(entityName: "Bookmark")
     
-    private func entity(for context: NSManagedObjectContext) -> NSEntityDescription {
-        return NSEntityDescription.entity(forEntityName: String(describing: Bookmark.self), in: context)!
+    override func setUp() {
+        super.setUp()
+        // Initialize sync so it will not fire on wrong thread.
+        _ = Sync.shared
```

Comment:
Sync should be initialized on main thread because it utilizes a webview for javascript calls.

Sometimes in tests first interaction with sync happened during a background thread call. Now we initialize sync before each tests to prevent threading problems